types/stripe-v3/index.d.ts

```diff
@@ -31,6 +31,7 @@ declare namespace stripe {
 
     interface StripeOptions {
       stripeAccount?: string;
+      betas?: string[];
```

Comment:
I couldn't find docs about this property in the page you mentioned. `StripeOptions` is used as a parameter of `StripeStatic` (which is documented here https://stripe.com/docs/stripe-js/reference#stripe-function)

Reply:
It is found under Checkout Client Quickstart and then step 3 ('write it yourself')
https://stripe.com/docs/payments/checkout/client#integrate

Reply:
Had a conversation with stripe developers on freenode #stripe and it IS part of the api as it stands even though it is not yet documented on https://stripe.com/docs/stripe-js/reference#stripe-function